canard/handler_list.h

```diff
@@ -85,16 +85,27 @@ class HandlerList {
         while (entry != nullptr) {
             if (transfer.data_type_id == entry->msgid &&
                 entry->transfer_type == transfer.transfer_type) {
-                entry->handle_message(transfer);
-                return;
+                if (entry->handle_message(transfer) &&
```

Comment:
I think the semantics of this return value are not good and don't match the original behavior. I think `handle_message` should return `true` if the handler is the final consumer of the message and `false` if other handlers should get a chance. Then the loop here also does not need to check the transfer type, it is implied by the handler.

Reply:
i changed it so the return value means that the message was for the right recipient